Frameworks/CoreFoundation/URL.subproj/CFURL.c

```diff
@@ -5007,21 +5007,35 @@ CFURLRef CFURLCreateCopyDeletingPathExtension(CFAllocatorRef allocator, CFURLRef
     }
 #endif
     
+    CFStringRef lastPathComponent = NULL;
```

Comment:
please put the comment at the start of the changed block. In this case that looks like it would be above the declaration for lastPathComponent

---
In reply to: [99227616](https://github.com/Microsoft/WinObjC/pull/1906#discussion_r99227616) [](ancestors = 99227616)